public/app/core/components/Select/MetricSelect.test.tsx

```diff
@@ -1,50 +1,52 @@
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
+import { select, openMenu } from 'react-select-event';
 
-import { LegacyForms } from '@grafana/ui';
+import { MetricSelect, Props } from './MetricSelect';
 
-import { MetricSelect } from './MetricSelect';
-
-const { Select } = LegacyForms;
+const props: Props = {
+  isSearchable: false,
+  onChange: jest.fn(),
+  value: '',
+  placeholder: 'Select Reducer',
+  className: 'width-15',
+  options: [
+    {
+      label: 'foo',
+      value: 'foo',
+    },
+    {
+      label: 'bar',
+      value: 'bar',
+    },
+  ],
+  variables: [],
+};
 
 describe('MetricSelect', () => {
-  describe('When receive props', () => {
-    it('should pass correct set of props to Select component', () => {
-      const props: any = {
-        placeholder: 'Select Reducer',
-        className: 'width-15',
-        options: [],
-        variables: [],
-      };
-      const wrapper = shallow(<MetricSelect {...props} />);
-      expect(wrapper.find(Select).props()).toMatchObject({
-        className: 'width-15',
-        isMulti: false,
-        isClearable: false,
-        backspaceRemovesValue: false,
-        isSearchable: true,
-        maxMenuHeight: 500,
-        placeholder: 'Select Reducer',
-      });
+  it('passes the placeholder, options and onChange correctly to Select', async () => {
+    render(<MetricSelect {...props} />);
+    const metricSelect = screen.getByRole<HTMLInputElement>('combobox');
```

Comment:
```suggestion
    const metricSelect = screen.getByRole('combobox');
```
I don't think providing type param is necessary here? Better infer them whenever possible.

Reply:
@Clarity-89 ah yes you're right, it's leftover from when i was playing with the `value` of the input - i'll remove 👍